gnocchi/storage/__init__.py

```diff
@@ -28,9 +28,14 @@
                default='file',
                help='Storage driver to use'),
     cfg.IntOpt('metric_processing_delay',
-               default=60,
+               default=30,
                help="How many seconds to wait between "
                "scheduling new metrics to process"),
+    cfg.IntOpt('tasks_per_worker',
+               default=256,
```

Comment:
256 seems high to be honest. this means worker1 will get 256 metrics in non-ceph drivers and if there are only 200 metrics in total, worker2+ will get nothing.

Reply:
You can fight with @akrzos on that but he tested a lot of values from 16 to 1024 and that's what worked best.

The problem is that anything under 256 metrics is barely enough to keeping a metricd busy for 30s, so when you're backlog is huge, metricd are idle 50% of the time. :(